4-star-nosed_mole/api_definitions/rest/ucs.yaml

```
@@ -101,20 +101,19 @@ paths:
           description: Multi-part upload successfully updated.
         "400":
           description: |
-            Cannot set invalid status for multi-part upload
-            with the given ID.
+            The provided status value was invalid.
```

We probably should further distinguish here.

This description just says, that we cannot change to the specific status, that was specified.
However, it could also be that the status currently or permanently can't be changed.

```suggestion
            The provided status value was invalid or the status of this multi-part upload currently or permanently can't be changed.
```